src/routes/[topicSlug]/[tableSlug]/[categorySlug].svelte

```diff
@@ -72,7 +112,27 @@
 </svelte:head>
 <BasePage>
   <span slot="header">
-    <DataHeader tableName={table ? table.name : null} location={locationName} {locationId} />
+    {#if showChangeAreaHeader}
+      <Header bind:showChangeAreaHeader showBackLink serviceTitle="Choose an area"
```

Comment:
Where does showBackLink come from?

Reply:
I am not sure if this is answering your question but `showBackLink` is the equivalent of the syntax `showBackLink=true`.

Reply:
👍